gulpfile.babel.js

```diff
@@ -112,7 +112,7 @@ gulp.task('build:search', ['build:status'], () => {
   fs.writeFileSync(searchFilePath, JSON.stringify(searchFeatures));
 });
 
-gulp.task('build:html', ['build:index', 'build:features', 'build:css'], () => {
+gulp.task('build:html', ['build:index', 'build:features', 'build:css'], () =>
```

Comment:
This form `() => something` is equivalent to this other `() => { return something; }` and not `() => { something }`. Are you sure you want to return the result of the `gulp.src(...)` of the next line?